torch/nn/attention/flex_attention.py

```diff
@@ -476,6 +485,33 @@ def _convert_mask_to_block_mask(
         return partial_blocks, None
 
 
+def or_masks(*mask_mods: _mask_mod_signature) -> _mask_mod_signature:
+    """Returns a mask_mod that's the union of provided mask_mods"""
+    if not all(callable(arg) for arg in mask_mods):
+        raise RuntimeError(f"All inputs should be callable mask_mods: {mask_mods}")
+
+    def or_mask(b, h, q_idx, kv_idx):
+        result = b.new_zeros((), dtype=torch.bool)
+        for mask in mask_mods:
+            result = result | mask(b, h, q_idx, kv_idx)
+        return result
+
+    return or_mask
+
+
+def and_masks(*mask_mods: _mask_mod_signature) -> _mask_mod_signature:
```

Comment:
should we check if the mod type is right?

Reply:
Meh, I actually don't really want to use that function much 🤔 I kinda want to remove it. I don't feel like checking the signature for how many positional args it has as a safety check is very pythonic 🤔